.travis.yml

```diff
@@ -36,14 +50,9 @@ before_deploy:
   - pip3 install --user docker-compose
   - docker --version
   - docker-compose --version
-  - if [ ! -d "$HOME/google-cloud-sdk/bin" ]; then rm -rf $HOME/google-cloud-sdk;
-    curl https://sdk.cloud.google.com | bash; fi
-  - source $HOME/google-cloud-sdk/path.bash.inc
-  - gcloud --quiet components install kubectl
-  - gcloud --quiet version
-  - openssl aes-256-cbc -K $encrypted_a1eb99cfc21e_key -iv $encrypted_a1eb99cfc21e_iv -in travis_secrets.tar.gz.enc -out travis_secrets.tar.gz -d
-  - tar -xzf travis_secrets.tar.gz
-  - gcloud auth activate-service-account --key-file service_key.json
+  - openssl aes-256-cbc -K $encrypted_555d9b2948d2_key -iv $encrypted_555d9b2948d2_iv
+    -in client_secrets.json.enc -out client_secrets.json -d
+  - gcloud auth activate-service-account --key-file client_secrets.json
```

Comment:
Hmm, alternatively, _maybe_ you could do something like:

```
openssl aes..... -d | gcloud auth activate-service-account --key-file /dev/stdin

```
?

Reply:
I think that will work...